### PR TITLE
add selenium test of temporarily broken IdP

### PR DESCRIPTION
**Pull request by lloyd**
This exercises code that displays an error message to users when their email provider goes offline without warning.

### CODE REVIEW

File: automation-tests/lib/wd-extensions.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ wd.prototype.wwin = function(opts, cb) {
 // optionally accepts waitForDisplayed opts object instead of CSS selector
 wd.prototype.wclick = function(opts, cb) {
   if (typeof opts === 'string') opts = { which: opts };
-  if (!opts.which) throw "css selector required";
+  if (!opts || !opts.which) throw "css selector required";
```
Comment (lloyd) [inline automation-tests/lib/wd-extensions.js:173]: just hardening of code.  an undefined first argument here would have caused a hard failure.  better would be a stack trace.

File: automation-tests/pages/css.js [hunk 2]
```diff
@@ -64,7 +64,8 @@ module.exports = {
     thirdEmail: '#email_2',
     signInButton: 'button#signInButton',
     notMyComputerButton: 'button#this_is_not_my_computer',
-    myComputerButton: 'button#this_is_my_computer'
+    myComputerButton: 'button#this_is_my_computer',
+    primaryOfflineCancel: '#primary_offline_confirm'
```
Comment (lloyd) [inline automation-tests/pages/css.js:68]: a new selector for the cancel button present in the screen that tells a user "Your email provider is broken right now"

**Comment by lloyd** [comment]
10/10 passes on vista_chrome:

```
$ scripts/run-all.js --platform vista_chrome -i 10 -t broken-primary
adding broken-primary.js
Running 10 suite(s) on 1 platform(s), 10 at a time against https://login.dev.anosrep.org
>>>>>>>>>>......................................................................................<....<...<.<...<..<...<...<..<...<

120/120 tests passed
```

File: automation-tests/lib/wd-extensions.js (hunk 1 above)
Comment (ozten) [inline automation-tests/lib/wd-extensions.js:173]: Nice.

File: automation-tests/tests/idp-transition/broken-primary.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    // primary is broken displayed!
+      .wclick(CSS['dialog'].primaryOfflineCancel)
+      // verify cancel button returns us to the initial screen
+      .wfind(CSS['dialog'].emailInput, done);
+  }
+},
+{
+  suiteName: path.basename(__filename),
+  cleanup: function(done) { testSetup.teardown(done) }
+});
```
Comment (ozten) [inline automation-tests/tests/idp-transition/broken-primary.js:108]: Awesome. Looks good.

**Pull request merged**